labs-docs.yml

```diff
@@ -17,7 +17,7 @@ content:
     branches: ['master']
     start_path: docs
   - url: https://github.com/neo4j-contrib/neo4j-streams
-    branches: ['4.0', '4.1']
+    branches: ['4.0']
```

Comment:
And what for people that have the previous versions?
I just want to fix the build, so that other things run again.

Reply:
The [Kafka module was removed](https://github.com/neo4j-documentation/labs-pages/pull/33) in December.  I will remove the reference from the Labs menu in neo4j-documentation/docs-ui#160.  The redirect is in place and `/labs/kafka/4.0` already redirects to `/docs/kafka` so that should be enough to be able to remove it completely from the build here.